code-samples/eventing/bookstore-sample-app/frontend/client/components/CommentDisplay.js

```diff
@@ -0,0 +1,22 @@
+const CommentDisplay = ({ comment }) => {
+	// Assume receiving a comment object
+	return (
+		<div className='flex my-4 p-4 justify-center'>
+			{/* Customized darkmode not working as expected yet */}
+			<div className='comment-display w-full w-7/12 flex flex-row rounded-lg p-4 bg-gray-800 text-white dark:bg-white'>
```

Comment:
Currently no matter in light mode or dark mode, the color of the comment box remains the same as this:
![CleanShot 2024-04-01 at 23 53 05@2x](https://github.com/knative/docs/assets/36619969/5ca0bfc6-2c8d-4dcd-b0be-579a3802a496)
![CleanShot 2024-04-01 at 23 53 02@2x](https://github.com/knative/docs/assets/36619969/f53b514d-974a-402a-bb36-3fe443f0ef4a)